### ISSUE TITLE
GatewayAPI

### ISSUE THREAD

**Issue by insxa** [AktuelSistem/WHMCS-SmsModule#102]
Hello, it works with WHMCS 7, can you integrate with this https://gatewayapi.com/docs/rest.html ?

**Comment by Oscah3** [comment]
Yes it does i'm also using InfoBip someone shared it with me here...

**Comment by Oscah3** [comment]
I need to show credit balance for infobip.com on the addon page but im not good with php

**Comment by emakundi** [comment]
InfoBip there is a function of showing balance. Just test it first with Postman and Postman can create the proper php script.

But what i do is different. I have created a cron script that send me an SMS everyday in the morning of my balance

**Comment by emir** [comment]
Hi,

GatewayAPI has been shut down, so this provider is no longer available. I am closing this issue.

We have recently started maintaining this open source project again. Good to connect here.

A new version has been released and it supports WHMCS 8.0 or later. Please check the latest release.

Best regards.
